src/main/java/org/gradle/wrapperupgrade/UpgradeWrapper.java

```diff
@@ -167,6 +167,10 @@ private void gitCommitAndPush(Params params, String commitMessage) {
     private void gitCreatePr(Params params, String prTitle, String prBody) throws IOException {
         if (!isDryRun()) {
             GHPullRequest pr = params.gitHub.getRepository(params.repository).createPullRequest(prTitle, params.prBranch, params.baseBranch, prBody);
+            List<String> labels = upgrade.getOptions().getLabels().get();
+            if (!labels.isEmpty()) {
+                pr.addLabels(labels.toArray(new String[0]));
+            }
```

Comment:
We don't have tests that creates real PRs on GitHub, were you able to test this manually?

Reply:
Yep, tested here 
👉  https://github.com/StefMa/wrapper-upgrade-gradle-plugin/pull/1

Published the plugin locally, then applied this patch 👇 
```diff
diff --git a/build.gradle b/build.gradle
index 2132948..7063682 100644
--- a/build.gradle
+++ b/build.gradle
@@ -7,14 +7,14 @@ plugins {
     id 'signing'
     id 'com.gradle.plugin-publish' version '1.3.0'
     id 'com.github.breadmoirai.github-release' version '2.5.2'
-    id 'org.gradle.wrapper-upgrade' version '0.11.4'
+    id 'org.gradle.wrapper-upgrade' version '99.99.99'
 }
 
 def releaseVersion = releaseVersion()
 def releaseNotes = releaseNotes()
 
 group = 'org.gradle'
-version = releaseVersion.get()
+version = "99.99.99"
 description = 'A Gradle plugin that detects and updates Gradle and Maven wrappers to the latest Gradle and Maven version.'
 
 repositories {
@@ -33,7 +33,10 @@ dependencies {
 wrapperUpgrade {
     gradle {
         'wrapper-upgrade-gradle-plugin' {
-            repo = 'gradle/wrapper-upgrade-gradle-plugin'
+            repo = 'stefma/wrapper-upgrade-gradle-plugin'
+            options {
+              labels = ["dependencies", "help wanted"]
+            }
         }
     }
 }
diff --git a/settings.gradle b/settings.gradle
index c52a48e..34ee2a0 100644
--- a/settings.gradle
+++ b/settings.gradle
@@ -1,3 +1,10 @@
+pluginManagement{
+    repositories {
+        gradlePluginPortal()
+        mavenCentral()
+        mavenLocal()
+    } 
+}
 plugins {
     id 'com.gradle.develocity' version '3.18.2'
     id 'com.gradle.common-custom-user-data-gradle-plugin' version '2.0.2'
```